packages/shared/src/hooks/supabase/queries/index.ts

```
@@ -0,0 +1,6 @@
+export { default as useMemosQuery } from './useMemosQuery';
+export { default as useMemoQuery } from './useMemoQuery';
+export { default as useCategoryQuery } from './useCategoryQuery';
+export { default as useSupabaseUserQuery } from './useSupabaseUserQuery';
+export { default as useSupabaseClientQuery } from './useSupabaseClientQuery';
+export { default as useSupabaseFeedbackClientQuery } from './useSupabaseFeedbackClientQuery';
```

ESLint

[error] Run autofix to sort these exports!

(simple-import-sort/exports)